### DISCUSSION TITLE
Wiz Motion Sensor

### DISCUSSION THREAD

**Discussion by ajaxodesa** [sbidy/wiz_light#181] [Ideas]
Hi,

I'm looking for the information about [Wiz Motion Sensor](https://www.wizconnected.com/en-US/consumer/products/046677603472/). Got one recently and trying to understand whether it can be more useful than with its limited native automation. Or maybe the product just too new and there is no documentation.

**Comment by sjee** [comment]
Someone [mentioned](https://github.com/sbidy/wiz_light/issues/46#issuecomment-753373563) having opened up one of those Wiz remotes, discovering an ESP 8266 based chip inside, which broadcasts wifi packets to ff:ff:ff.  So the lights probably listen to this. My guess is the motion sensors works similar to the remotes. 
_(I've not seen anything in the API documentation about the remotes or the motion sensors.)_

I might try a (more expensive) [Shelly Motion](https://shelly.cloud/shelly-motion-smart-home-automation-sensor/) or the (cheaper) [DIY route](https://peyanski.com/cheap-diy-motion-sensor-esphome-home-assistant/) at some point.

**Comment by XRyu** [comment]
You might try pondering with the ESP, flashing it with ESPHome directly.
ESP Home is pretty easy to use. If i had one i would have tried that route
and adding an Recipe for ESP Home :)

Am Mo., 11. Okt. 2021 um 16:50 Uhr schrieb sjee ***@***.***>:

**Reply by sjee** [reply]
I will try with ESP Home. My local electronics store does not have the parts so I will probably have to import them. (That will be a long wait.)

### Workaround: using a Philips Hue motion sensor + Wiz Lights

For now I have connected two Wiz lights to a Philips Hue motion sensor I had lying around. I created a flow in the Node-RED integration of Home Assistant. I've set it up so the flow only gets triggered during a particular time of day and when no motion is detected the flow turns the lights off again after three minutes.

One little caveat when using a Philips Hue motion sensor: one **must** turn on that motion sensor in the Philips Hue app and the app requires the motion sensor to be connected to at least one light. Otherwise HA will not receive a state change. 
The light that is connected to the motion sensor via the Philips Hue app will also turn on, but this can be solved by setting the "daylight behaviour" and "night behaviour" of the motion sensor to "Do nothing" (in the Philips Hue app).

**Comment by blkrx7** [comment]
I've got ESP Home setup on both a wiz remote, and wiz motion sensor.
Unfortunately the ESP isn't woken up until there is motion, and it's shut
off before esphome initializes and connects to the network. I haven't had a
lot of time to tinker with it, but there is going to need to be some
hardware changes to get esphome working properly.

Hoping to find some time this winter to look at it again.
